src/main/webapp/index.html

```diff
@@ -19,7 +16,7 @@
     
     <title>Mintern</title>
   </head>
-  <body>
+  <body onload="fetchLogin()">
```

Comment:
Consider that we will eventually have a function to load all needed first things and you will probably have to call the fetchLogin() function from there (e.g. loadEverything()). Just as a reminder for when we merge everything into the master.

Reply:
Yup, I did have it in mind! I was waiting for either one of us to make push first, so the second one incorporates it into said `loadEverything()` function. To avoid merging conflicts, I think it would be better for you to pull from master after this is pushed to master, and you just move this function.

Reply:
Sounds great!